src/WebJobs.Script.Scaling/IScaleHandler.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Scaling
+{
+    [SuppressMessage("Microsoft.Naming", "CA1711:IdentifiersShouldNotHaveIncorrectSuffix", Justification = "By design")]
+    public interface IScaleHandler
```

Comment:
Add more comment.

```csharp
    /// <summary>
    /// this is implemented by core function runtime and it will be called by ScaleManager
    /// </summary>
    [SuppressMessage("Microsoft.Naming", "CA1711:IdentifiersShouldNotHaveIncorrectSuffix", Justification = "By design")]
    public interface IScaleHandler
```